virttest/qemu_devices/qcontainer.py

```diff
@@ -2108,6 +2110,12 @@ def define_hbas(qtype, atype, bus, unit, port, qbus, pci_bus, iothread,
             devices.append(protocol_node)
             devices.append(format_node)
             # Add filter node
+            if image_copy_on_read:
```

Comment:
if image_copy_on_read in ("yes", "on", "true")

Reply:
Have updated, thanks zhenchao.